src/components/fields/image-upload/image-upload.tsx

```diff
@@ -120,9 +118,9 @@ export const ImageUploadInner = (props: IGenericFieldProps<IImageUploadSchema>)
 				.test(
 					"max-size",
 					maxFileSizeRule?.errorMessage ||
-						ERROR_MESSAGES.UPLOAD("photo").MAX_FILE_SIZE(maxFileSizeRule?.["maxSize"]),
+						ERROR_MESSAGES.UPLOAD("photo").MAX_FILE_SIZE(maxFileSizeRule?.["maxSizeInKb"]),
 					(value) => {
-						if (!value || !Array.isArray(value) || !maxFileSizeRule?.["maxSize"]) return true;
+						if (!value || !Array.isArray(value) || !maxFileSizeRule?.["maxSizeInKb"]) return true;
 						return value.reduce((accumulator, file) => {
 							if (
 								!accumulator ||
```

Comment:
- rename test name from `max-size` to `max-size-in-kb` for consistency
- missed out renaming `maxFileSizeRule?.["maxSizeInKb"]` on line 127